proxy/http/remap/RemapPlugins.h

```diff
@@ -38,17 +38,12 @@
  * A class that represents a queue of plugins to run
  **/
 struct RemapPlugins : public Continuation {
-  RemapPlugins() : _cur(0), _rewriten(0) {}
+  RemapPlugins() = default;
   RemapPlugins(HttpTransact::State *s, URL *u, HTTPHdr *h, host_hdr_info *hi)
-    : _cur(0), _rewriten(0), _s(s), _request_url(u), _request_header(h), _hh_ptr(hi)
+    : _s(s), _request_url(u), _request_header(h), _hh_ptr(hi)
   {
   }
 
-  ~RemapPlugins() override
-  {
-    _cur      = 0;
-    _rewriten = 0;
```

Comment:
Where these reset before for some reason? I agree that it seems odd, wondering why they bothered in the first place ...